client/.eslintrc.js

```diff
@@ -26,15 +26,22 @@ module.exports = {
     "no-lonely-if": "off",
     "consistent-return": "off",
     "no-control-regex": "off",
+    'no-multiple-empty-lines': 'warn',
     "no-script-url": "off", // some <a> tags should have href="javascript:void(0)"
+    'react/button-has-type': 'warn',
+    'react/destructuring-assignment': 'warn',
```

Comment:
I dunno if I like this one @koooge. Do think this rule is valuable?

Reply:
I fixed one `react/button-has-type` error and delete this warning rule 👌  Basically, I think we should obey the recommended rule.
But there are many (164) `react/button-has-type` warnings. It may be annoying..

Reply:
I was actually referring to line 32 `'react/deconstructing-assignment': 'warn'`.

Reply:
Oops, I supposed we should obey `react/deconstructing-assignment` rule "in basically", too. But now I disabled this rule to obey the current code base.